--- Linux-FBDev-kernel-src_2.6.00.02a/Makefile_26_Lite.kernel.old	2006-10-08 19:40:23.000000000 +0200
+++ Linux-FBDev-kernel-src_2.6.00.02a/Makefile_26_Lite.kernel	2006-10-08 19:41:07.000000000 +0200
@@ -2,6 +2,6 @@
 # Makefile for the VIA framebuffer driver (for Linux Kernel 2.6)
 #
                                                                                 
-obj-$(CONFIG_FB_VIA) += viafb.o
+obj-m += viafb.o
 
-viafb-objs := viafbdev.o hw.o iface.o tv.o via_i2c.o vt1622.o vt1622a.o dvi.o lcd.o vt1625.o ioctl.o accel.o
+viafb-objs := viafbdev.o hw.o iface.o tv.o via_i2c.o vt1622.o vt1622a.o dvi.o lcd.o vt1625.o ioctl.o accel.o IntegratedTV.o via_utility.o
